ProcessMaker/Model/Process.php

```diff
@@ -166,7 +167,7 @@ class Process extends Model
      */
     protected $rules = [
         'uid' => 'max:36',
-        'name' => 'required',
+        'name' => 'required|unique:processes,name',
```

Comment:
Does the name need to be unique? That is not a requirement.

Reply:
changed.